From c2f6b2bdd26f3a17c572e2d220ec27d47fa6c695 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Tue, 4 Sep 2018 16:41:48 +0200
Subject: [PATCH 11/35] nile-common: sepolicy: Label sys.listeners.registered

* QCOM has moved to `vendor.sys.listeners.registered` but our
  blobs still use `sys.listeners.registered`.

Change-Id: I0a51536fc86d7e82fa181d9844b7d4e140cf9087
---
 sepolicy/vendor/property_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sepolicy/vendor/property_contexts b/sepolicy/vendor/property_contexts
index 0ffeecf..e4e7519 100644
--- a/sepolicy/vendor/property_contexts
+++ b/sepolicy/vendor/property_contexts
@@ -5,3 +5,4 @@ persist.net.doxlat               u:object_r:vendor_net_radio_prop:s0
 persist.tareset.notfirstboot     u:object_r:tareset_notfirstboot_prop:s0
 semc.version.                    u:object_r:semc_version_prop:s0
 sony.modemswitcher               u:object_r:sony_modemswitcher_prop:s0
+sys.listeners.registered         u:object_r:vendor_tee_listener_prop:s0
-- 
2.17.1

